releasenotes/notes/0.18/add-initialize-from-method-to-schedules-ce86083b5bd39e6b.yaml

```diff
@@ -0,0 +1,15 @@
+---
+features:
+  - |
+    A new constructor method
+    :meth:`~qiskit.pulse.Schedule.initialize_from` ws added to the
+    :class:`~qiskit.pulse.Schedule` and :class:`~qiskit.pulse.ScheduleBlock`
+    classes. This method initializes a new empty schedule which
+    takes the attributes from other schedule. For example:
```

Comment:
Do we know which attributes are carried over? `name` is in the example, but is it all attributes on the `Schedule`?

Reply:
It looks like it's just `name` and `metadata`:  https://github.com/Qiskit/qiskit-terra/blob/main/qiskit/pulse/schedule.py#L170-L178 and for schedule block it also adds `alignment_context`: https://github.com/Qiskit/qiskit-terra/blob/main/qiskit/pulse/schedule.py#L963-L977